src/pages/saksbehandling/steg/beregningOgSimulering/beregning/FradragInputs.tsx

```diff
@@ -149,6 +149,69 @@ export const fradragSchema = yup.object<FradragFormData>({
         .defined(),
 });
 
+const TilOgMedRadios = (props: {
+    radioValue?: Nullable<Date>;
+    beregningStartDato?: Nullable<Date>;
+    beregningSluttDato?: Nullable<Date>;
+    periodeFraOgMed?: Nullable<Date>;
+    periodeName: string;
+    onFradragChange: (date: Date) => void;
+    intl: IntlShape;
+}) => {
+    const { periodeFraOgMed, beregningStartDato, beregningSluttDato } = props;
+
+    if (!periodeFraOgMed || !beregningStartDato || !beregningSluttDato) {
+        return null;
+    }
+
+    const forrigeKalendermåned = DateFns.subMonths(DateFns.endOfMonth(new Date()), 1);
+
+    const getTilOgMedDatoer = (startDate?: Nullable<Date>, endDate?: Nullable<Date>) => {
+        if (!startDate || !endDate) {
+            return [];
+        }
+
+        const dateInterval = DateFns.eachMonthOfInterval({
+            start: startDate,
+            end: endDate,
+        });
+
+        return dateInterval.map((date) => DateFns.endOfMonth(date));
+    };
```

Comment:
IMO hadde det vært lettere å henge med dersom det  var sånn:
```suggestion
    const datoer = DateFns.eachMonthOfInterval({
        start: beregningStartDato,
        end: forrigeKalendermåned
        })
        .map((d) => DateFns.endOfMonth(d));
```

(da gitt at det blir fikset sånn at det ikke er lov å sende inn ugyldige verdier til denne komponenten).